packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx

```diff
@@ -53,13 +55,13 @@ const useStyles = makeStyles((theme: Theme) => ({
     height: '3rem',
   },
 
-  validityLabel: {
+  validityLabelRoot: {
     color: theme.palette.error.main,
```

Comment:
Also for the color you can use the `MUI Typography color prop` avoiding the class creation! Have a look:
`<Typography color="primary | secondary | textPrimary ... or even error">Foo bar</Typography>`